src/components/vue-form-renderer.vue

```diff
@@ -44,24 +44,25 @@
 import Vue from 'vue';
 import * as VueDeepSet from 'vue-deepset';
 import debounce from 'lodash/debounce';
-import { HasColorProperty, shouldElementBeVisible, getValidPath } from '@/mixins';
+import {getValidPath, HasColorProperty, shouldElementBeVisible} from '@/mixins';
```

Comment:
Should we return the spacing between the braces in this file?

Reply:
Yeah, it could be a setting that's missing in eslint or my config for this project. Let me take a look.